usr/share/lib/img_proof/tests/SLES/GCE/test_sles_gce_sev.py

```diff
@@ -0,0 +1,4 @@
+def test_sles_gce_sev(host):
+    expected = 'AMD Secure Encrypted Virtualization (SEV) active'
```

Comment:
Let's see how long this let's, maybe we get lucky and the message will stick. If not we can simplify and look for "SEV" and "active" rather than matching the hole string.

Reply:
Right, if we want to make it simpler now that seems good but it's nice to be as expressive as possible without causing us any false negatives. Since the command is already grepping on SEV we really only need to expect `active` to be in the output.